api/src/main/java/com/datastrato/gravitino/Catalog.java

```diff
@@ -88,6 +89,34 @@ enum CloudName {
    */
   String CLOUD_REGION_CODE = "cloud.region-code";
 
+  /**
+   * This variable is used as a key in properties of catalogs to use authorization plugin provider
+   * in Gravitino. <br>
+   * You can use your own authorized provider with this configuration The object you use may be
+   * unstable, so don't use it unless you know what you're doing.
+   */
+  String AUTHORIZATION_PLUGIN = "authorization-plugin";
```

Comment:
```
   * You can use your own authorized provider with this configuration The object you use may be
   * unstable, so don't use it unless you know what you're doing.
```

What is the meaning of this description, what are you trying to emphasize?

Reply:
Improvement of these code comments.